psycopg3/setup.py

```diff
@@ -32,7 +32,7 @@
         f"psycopg3-binary == {version}",
     ],
     "test": [
-        "mypy >= 0.812",
+        "mypy >= 0.812 ; implementation_name!='pypy'",
```

Comment:
Please use consistent spacing around the operator

Reply:
removed space before ';'

Reply:
I meant the spaces around the operator, as you used in the other line :)